### ISSUE TITLE
[A11y] Mobile Work Show Page-CPD Child: Mobile View Size 320

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1956]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Child: https://staging.oregondigital.org/concern/documents/df727057n?locale=en

For some reason the mobile viewport for 320 is off. See the screen snap. The content is half the size of what 320 should be.

<img width="241" alt="Screen Shot 2021-08-23 at 8 46 40 AM" src="https://user-images.githubusercontent.com/8353860/130477641-076be0de-f03f-4090-b157-8ebc847489ad.png">

### Suggested Fix

Fix the width so its using the full 320 view.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
Fixed by #2162

**Comment by azle** [comment]
@CGillen  Looks good on mobile, can we just add some left and right spacing for the text that appears in the "Is part of" section. The checkbox and thumbnail looks good but the text below it needs the same spacing as the tab content above as well as the save button on the right. I am guessing its about 5 or 10px left and right. See screen snap.

I also noticed that, with the long links its scrolling horizontally. Do we want to create a fix width so it wraps instead and the user only has access to vertical scrolling.

![IMG_5081](https://user-images.githubusercontent.com/8353860/150395909-a9b9dead-e518-4a3c-84fd-f29db64456c0.PNG)
